PowerEditor/installer/nativeLang/danish.xml

```diff
@@ -918,6 +920,7 @@
             <NbFileToOpenImportantWarning title="Du er ved at åbne et stort antal filer" message="$INT_REPLACE$ filer er ved at blive åbnet.\rEr du sikker på, at du vil åbne dem?"/>
 			<SettingsOnCloudError title="Indstillinger i skyen" message="Det ser ud til at stien af indstillinger i skyen er sat på et skrivebeskyttet drev,\reller på en mappe som kræver priviligerede rettigheder for at få skriveadgang.\rDine indstillinger på sky annulleres. Nulstil venligst en relateret værdi i dialogen Præferencer."/>
 			<FilePathNotFoundWarning title="Åbning af fil" message="Filen du prøver at åbne findes ikke."/>
```

Comment:
All the '\r' should be replaced by real carriage return or be removed.
Please see https://github.com/notepad-plus-plus/notepad-plus-plus/commit/193ecfd6dfaa2c37e0d9aa5604bed7568a6c316f